src/LibraryViewExtension/web/library/layoutSpecs.json

```diff
@@ -388,10 +388,6 @@
                 },
                 {
                   "path": "DSCoreNodes.DSCore.DateTime.Format"
-                },
-                {
-                  "path": "DSCoreNodes.DSCore.DayOfWeek",
-                  "iconUrl": "./dist/resources/DSCore.DayOfWeek.png"
                 }
```

Comment:
More resource cleanup.

Reply:
Again, no idea why this its to be removed.